docs/jwst/pathloss/description.rst

```diff
@@ -6,23 +6,30 @@ Overview
 --------
 
 The pathloss correction step calculates the correction to apply to spectra
-when the 1-d extraction is performed.  This correction accounts for losses
+when the 1-d extraction is performed.  The motivation behind the correction
+is different for NIRSPEC and NIRISS, while that for MIRI has not been
+implemented in Build 7.3.  For NIRSPEC, this correction accounts for losses
 in the optical system due to light being scattered outside the grating, and
-to light not passing through the aperture.
+to light not passing through the aperture, while for NIRISS_SOSS data it
```

Comment:
Minor nit, but you don't need the underscore in NIRISS_SOSS. They can stand as 2 independent acronyms.

Reply:
OK